fluxacademy.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<head>
+	<title>Flux Academy</title>
```

Comment:
extra space before `<title>`
a good way to see if something is actually indented properly is to make sure that it aligns with the light gray vertical line in your editor; you should always be checking for this